_includes/payment-order-other-features.md

```diff
@@ -1093,6 +1269,17 @@ The structure of a problem message will look like this:
 
 {% include merchant-authenticated-consumer.md %}
 
+## One-click Payments
+
+One-Click Payments should be used if you want to present your own Payment Menu 
+in all channels you want to support. This means that if you are both on the 
+web and in-app you will need to build your own payment menu to be able to use 
+the One-Click Payment functionality. You will need to ask the system to generate
+a `paymentToken` (with `generatePaymentToken` in a `Purchase` or `Verify`
+request) to be able to use this feature. You will also need to turn the 
+`Payment Order` into instrument mode (only show one instrument) 
+with the input parameter instrument.
```

Comment:
The `instrument` word needs to link to the description of that feature once it gets added in DX-1016.

```suggestion
with the request parameter `instrument`.
```